ChangeLog

```diff
@@ -0,0 +1,6 @@
+Current - TBD
+
+	- `--thumbside` renamed to `--thumbsize`
+
+
+0.4 - ?
```

Comment:
```suggestion
```

From readme:

```
We don't maintain the ChangeLog file (in the GNU compatible format), since
`git log` does just the same job.
```